src/app/books/book/book.component.html

```diff
@@ -0,0 +1,11 @@
+<mat-card class="mat-elevation-z4">
+  <mat-card-title-group>
+    <mat-card-title>{{ book.title }}</mat-card-title>
```

Comment:
To be absolutely sure the app wouldn't break if there is no input value for book, write: book?.title, book?.book_image, book?.description...